deploy/rbac.yaml

```diff
@@ -59,6 +59,9 @@ rules:
   - list
   - watch
   - create
+  - update
+  - patch
+  - delete
```

Comment:
this needs to be fixed in CSV as well - https://github.com/openshift/local-storage-operator/blob/master/manifests/4.8/local-storage-operator.v4.8.0.clusterserviceversion.yaml

Reply:
👍🏽  🙏🏽